src/components/Carousel/Carousel.css

```diff
@@ -0,0 +1,72 @@
+.carousel {
+    width: 575px;
+    height: 324px;
```

Comment:
For some reason the carousel is rendering at 334px height, and some of the internal dimensions seem off.  For instance the title is supposed to be level with the top of the image. This doesn't need to be fixed before merge, since this is a prototype. 

![image](https://user-images.githubusercontent.com/37159177/110391028-e6934d00-8034-11eb-8219-05692f7c66cc.png)

Reply:
Hmm, they aren't even in figma? I had assumed it was meant to be vertically centered but can look in finer detail later ![image](https://user-images.githubusercontent.com/48805474/110497994-c8b4ff00-80ab-11eb-92f4-97c87dcb7d28.png)
![image](https://user-images.githubusercontent.com/48805474/110498082-de2a2900-80ab-11eb-91a1-66617d58646c.png)

Also, this is what I was referring to in Slack when I said this PR would not cover styling and had everything set as position absolute as a placeholder. My plan was to come back to that in a separate PR or I can commit updates this evening if we want to hold on merging. May be good to get it in though to sort merge conflicts and avoid more. Thoughts?

Reply:
Yeah, you're right. I guess I looked at the figma wrong. You can ignore this.